types/values.d.ts

```diff
@@ -0,0 +1,3 @@
+import { ValueOfUnion } from './util/tools';
+
+export function values<T extends object, K extends keyof T>(obj: T): Array<T[K] | ValueOfUnion<T>>;
```

Comment:
The following is typescript official type of `Object.values`, maybe it is more practical:

```ts
    /**
     * Returns an array of values of the enumerable properties of an object
     * @param o Object that contains the properties and methods. This can be an object that you created or an existing Document Object Model (DOM) object.
     */
    values<T>(o: { [s: string]: T } | ArrayLike<T>): T[];
```